docs/tutorials/gluon/gluon_from_experiment_to_deploymen.md

```diff
@@ -0,0 +1,400 @@
+# Gluon: from experiment to deployment, an end to end example
+
+## Overview
+
+MXNet Gluon API comes with a lot of great features and it can provide you everything you need from experiment to deploy the model.
+In this tutorial, we will walk you through a common used case on how to build a model using gluon, train it on your data, and deploy it for inference.
+
+Let's say you want to build a service that provides flower species recognition. A common use case is, you don't have enough data to train a good model like ResNet50.
+What you can do is utilize pre-trained model from Gluon, tweak the model according to your neeed, fine-tune the model on your small dataset, and deploy the model to integrate with your service.
+
+We will use the [Oxford 102 Category Flower Dateset](http://www.robots.ox.ac.uk/~vgg/data/flowers/102/) as an example to show you the steps.
+
+## Prepare training data
+
+You can use this [script](https://github.com/Arsey/keras-transfer-learning-for-oxford102/blob/master/bootstrap.py) to download and organize your data into train, test, and validation sets. Simply run:
```

Comment:
please move this scripts to the `docs/tutorial_utils` folder, modify it so that it has a `get_data(dirname=)` function and use `mx.gluon.utils.download()` to download it, import it and execute it. The tutorial should be able to run end to end in the CI for test verification.

Reply:
I can change that for python part. Wondering how the C++ inference code should be tested. I am thinking just show code snippets here and leverage [C++ examples in nightly test](https://github.com/apache/incubator-mxnet/blob/master/cpp-package/tests/ci_test.sh). What do you think?

Reply:
Yes I think that's the best strategy with our current setup :+1: